### PR TITLE
create module, rename class, fix layout banner, brand

### PR DESCRIPTION
**Pull request by thanhtrantruong47**

### CODE REVIEW

File: practice-two/src/index.html [hunk 1]
```diff
@@ -47,193 +47,189 @@ <h1><a class="link" href="javascrip:void(0)"><img class="navbar__logo" src="./as
 
   <!-- banner start  -->
   <section class="banner">
-    <div class="banner__item">
+    <div class="banner__box">
       <div class="banner__desc">
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/index.html:51]: banner__content

File: practice-two/src/index.html [hunk 2]
```diff
@@ -47,193 +47,189 @@ <h1><a class="link" href="javascrip:void(0)"><img class="navbar__logo" src="./as
...
   </section>
   <!-- banner end  -->
 
   <!-- brand start  -->
   <section class="brand">
-    <div class="brand__item">
-      <img src="./assets/images/brands/versace.png" alt="versace">
-      <img src="./assets/images/brands/zara.png" alt="zara">
-      <img src="./assets/images/brands/gucci.png" alt="gucci">
+    <div class="brand__box container">
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/index.html:73]: brand__wrapper

File: practice-two/src/styles/bases/base.scss [hunk 3]
```diff
@@ -5,4 +5,5 @@
 html {
   font-family: Integral CF, Satoshi;
   font-size: $size;
+  max-width: 1440px;
```
Comment (tuanthaiquang-agilityio) [inline practice-two/src/styles/bases/base.scss:8]: Why do we need max-width? What would the UI look like if it were larger than 1440px? Will there be white space on the left and right sides?

File: practice-two/src/styles/layouts/brand.scss [hunk 4]
```diff
@@ -23,13 +22,16 @@
...
+
+  .brand--center {
+    display: none;
   }
 
   .brand__img {
+    display: none;
+  }
+
+  .brand__img-desktop {
```
Comment (tuanthaiquang-agilityio) [inline practice-two/src/styles/layouts/brand.scss:34]: why do we need `-desktop` in class name?

File: practice-two/src/styles/modules/link.scss [hunk 5]
```diff
@@ -6,5 +6,6 @@
 .link-primary {
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/styles/modules/link.scss:6]: Why do you need a link-primary class? Can we use the link class only?

File: practice-two/src/styles/modules/desc.scss [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+@media (min-width : 1440px) {
+  .desc {
+    font-size: toRem(16px);
+
+    &--primary {
+      font-size: toRem(16px);
+    }
+
+    &--secondary {
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/styles/modules/desc.scss:25]: Can you remove it?

**Pull request merged**